### PR TITLE
Xamarin AzureMessaging GA for new APIs

### PR DESCRIPTION
**Pull request by mpodwysocki**
This PR will update the AzureMessaging Xamarin libraries to 3.0 for Apple and 1.0 for Android.  Apple support now includes tvOS, macOS and iOS.

### CODE REVIEW

File: XPlat/AzureMessaging/iOS/source/ApiDefinition.cs [hunk 1]
```diff
@@ -166,6 +167,9 @@ public partial interface MSNotificationHub
 		[Static, Export("startWithConnectionString:hubName:")]
 		void Start(string connectionString, string hubName);
 
+		[Static, Export("StartWithInstallationManagement:")]
+		void StartWithInstallationManagement(MSInstallationManagementDelegate managementDelegate);
```
Comment (mattleibow) [inline XPlat/AzureMessaging/iOS/source/ApiDefinition.cs]: It might just be better to call this `Start` and have it as a simple overload?

Comment (mpodwysocki) [reply XPlat/AzureMessaging/iOS/source/ApiDefinition.cs]: Renamed to start

**Comment by mattleibow** [comment]
Seems CI is giving a 404 when downloading the android bits?

**Comment by mpodwysocki** [comment]
> Seems CI is giving a 404 when downloading the android bits?

Correct @mattleibow, that's why it's in draft right now because we're waiting on the Android release.  Once it has been released, I'll move it to active.

**Comment by mpodwysocki** [comment]
@mattleibow not sure why CI seems to be failing, but doesn't seem related to any changes I've made

**Review by mattleibow — APPROVED** [review]

**Pull request merged**